bin/todo

```diff
@@ -18,17 +18,21 @@ import re
 import sys
 import shutil
 
+# python2 compat
+if sys.version_info[0] == 2:
+    from future.builtins.misc import input
+
 import todo_tools
 
 
 def main():
     args = get_args()
 
-    # python 2 compatibility
-    if sys.version_info[0] == 2:
-        input = raw_input
-
-    if args.check:
+    if args.clear_todos:
+        sure = input("Are you sure you want to clear todo file? (y/N) ").lower()
+        if sure == 'y':
+            open(args.file, 'w').close()
+    elif args.check:
```

Comment:
I don't have a formed opinion on this, but are you sure we want a clear-todos option?

I was initially thinking people would just edit their `~/.todos` file, since it's really legible, and each line corresponds to a todo.

Is clearing all todos something we think people will do a lot?